contrib/mesos/pkg/executor/executor.go

```diff
@@ -206,10 +206,23 @@ func (k *Executor) isDone() bool {
 }
 
 // sendPodUpdate assumes that caller is holding state lock; returns true when update is sent otherwise false
-func (k *Executor) sendPodUpdate(u *kubetypes.PodUpdate) bool {
+func (k *Executor) sendPodUpdate(sendSnapshot bool) bool {
 	if k.isDone() {
 		return false
 	}
+	var snapshot []*api.Pod
+	if sendSnapshot {
+		snapshot = make([]*api.Pod, 0, len(k.pods))
```

Comment:
This function feels strange. It used to send an update as the name suggests. Why was this changed? You can call the old function with 
```
&kubetypes.PodUpdate{
		Op:   kubetypes.SET,
		Pods: []*api.Pod{},
	}
```